src/index.tsx

```diff
@@ -1,12 +1,65 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import { isNotDefined } from '@togglecorp/fujs';
+import { isNotDefined, isDefined } from '@togglecorp/fujs';
+import * as Sentry from '@sentry/react';
+import {
+    useLocation,
+    useNavigationType,
+    createRoutesFromChildren,
+    matchRoutes,
+} from 'react-router-dom';
+import {
+    appTitle,
+    appCommitHash,
+    api,
+    environment,
+    sentryAppDsn,
+    sentryReplaysOnErrorSampleRate,
+    sentryReplaysSessionSampleRate,
+    sentryTracesSampleRate,
+} from '#config';
 
 import 'mapbox-gl/dist/mapbox-gl.css';
 import './index.css';
 
 import App from './App/index.tsx';
 
+if (isDefined(sentryAppDsn)) {
+    Sentry.init({
+        dsn: sentryAppDsn,
+        release: `${appTitle}@${appCommitHash}`,
```

Comment:
Lets try to add version as well (if possible)

Reply:
Added `version` as well.